rtl/eclrtl/rtlrecord.cpp

```diff
@@ -134,6 +135,8 @@ class IfBlockInfo
     const RtlFieldInfo &field;
     const IfBlockInfo *parent = nullptr;  // for nested ifblocks
     unsigned idx;
+public:
+    unsigned startIdx; // For ifblocks inside child records
```

Comment:
better if this was private, but may not be worth it.

Reply:
Agree, will upload a fix shortly